website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -363,10 +363,11 @@ az devops security permission update --organization https://dev.azure.com/<org_n
 
 You must connect your service user before setting up a dbt Cloud project, as the service user's permissions determine which projects dbt Cloud can import.
 
-A dbt Cloud account admin with access to the service user's Azure DevOps account must complete the following to connect the service user:
-1. Sign in to the service user's Azure DevOps account.
-2. In dbt Cloud, click **Link Azure Service User**.
-3. You will be directed to Azure DevOps and must accept the Microsoft Entra ID app's permissions.
+When using Azure AD to SSO to connect to dbt Cloud, and also SSO to your MS tools, a dbt Cloud account admin with access to the service user's Azure DevOps account must complete the following to connect the service user:
```

Comment:
Based on Benoit's feedback [here](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1726656354040079), we need to revamp the original instructions (rather than remove them) and then add this section as sort of an FAQ if they have already set up the integration, but it's not working properly. Let me know if you have any questions!

Reply:
Thanks @matthewshaver